esp/esp/program/modules/handlers/jsondatamodule.py

```diff
@@ -468,6 +475,7 @@ def class_info(self, request, tl, one, two, module, extra, prog):
     class_info.cached_function.depend_on_model(ClassSection)
 
     @aux_call
+    @no_auth
```

Comment:
I think it doesn't matter because the definition of `@json_response` uses `functools.wraps`. I'll change it for consistency.